app/questionnaire/router.py

```diff
@@ -75,26 +75,26 @@ def get_next_location_url(self, location, routing_path, return_to_summary=False)
         is_section_complete = self._progress_store.is_section_complete(
             location.section_id, location.list_item_id
         )
+        if return_to == "summary" and is_section_complete:
+            return self._get_section_url(location)
+
+        if return_to == "final_summary" and is_section_complete:
```

Comment:
It is best practice to use hyphens in URLs over underscores.

Reply:
I used to be a web developer for 7 years, fancy doing that, I wonder what else I have forgotten